src/App.jsx

```diff
@@ -1,107 +1,29 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import usersFromServer from './api/users.json';
+import commentsFromServer from './api/comments.json';
+import { PostList } from './components/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId)
+    || null;
+}
+
+function getCommentById(id) {
+  // return commentsFromServer.filter(comment => postId.includes(comment.postId));
+  return commentsFromServer.filter(comment => comment.postId === id);
```

Comment:
The commented code on line 17 can be removed to keep the codebase clean and avoid confusion in the future.